### ISSUE TITLE
[BUG][a11y] Secondary grey text colour is not accessible 

### ISSUE THREAD

**Issue by Qrzy** [DivanteLtd/storefront-ui#1065]
This particular a11y issue is rather related to the designs.
As reported here: https://github.com/DivanteLtd/next/issues/389 the secondary colour is at too low contrast with white background and does not pass WCAG check.
@mbaran-dvnt would you kindly take a closer look?

**Comment by mbaran-dvnt** [comment]
Hi guys,

I've gone through all of the designs, fixed the grey colors in the style guide and I hope I've changed every state on the pages. I've also fixed some of the font size issues (text button). All should be now aligned with a11y :)

**Comment by martaradziszewska** [comment]
Fixed in #1121
